src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -3769,9 +3769,11 @@ def spawn(self, launcher, cu):
         with open(launch_script_name, "w") as launch_script:
             launch_script.write('#!/bin/bash -l\n\n')
 
-            launch_script.write("echo script start_script `%s` >> %s/PROF\n" % (cu['gtod'], cu_tmpdir))
+            if 'RADICAL_PILOT_PROFILE' in os.environ:
```

Comment:
> FWIW: the profile itself also depends on the env var, and that seemed to work reliably in sub-agents so far...

What do you mean with this? We were talking about the profile(r) all the time already?